app-web/__tests__/components/CardCarousel.test.js

```diff
@@ -0,0 +1,67 @@
+import React from 'react';
+import { render, cleanup, fireEvent } from 'react-testing-library';
+import 'jest-dom/extend-expect';
```

Comment:
you can use setupFilesAfterEnv for import of jest-dom/extend-expect for all tests